src/views/Chat/MestringChat.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react';
+import ChatBody from './Sections/ChatBody';
+import ChatHeader from './Sections/ChatHeader';
+import ChatInput from './Sections/ChatInput';
+import { ISocketMessage, ITextMessage } from '../../interfaces/IMessage';
+import '../../styles/Chat.less';
+import { createEnterMestringQueueMessage } from '../../services/message-service';
+
+const MestringChat = () => {
```

Comment:
Har gått over på å gjøre det likt i student frontenden som vi gjør i frivillig frontenden, med å bruke context apiet og reducers til å holde på meldingsstate gjennom hele applikasjonen, sånn at eleven kan navigere rundt på siden uten å bli kastet ut av køen fra frontenden sin side.

Se hvordan det er gjort i Chat.tsx i cleanup-chat branchen for å se hvordan du gjør det på samme måte i MestringChat komponenten.